docs/development/design/apiv3.rst

```diff
@@ -1,15 +1,34 @@
-Design of APIv3
-===============
+APIv3
```

Comment:
I think we should include design in the title still, so it's easier to find in search, etc.